callstats-java-sdk/src/main/java/io/callstats/sdk/httpclient/CallStatsHttpClient.java

```diff
@@ -104,19 +106,25 @@ public CallStatsHttpClient() {
 		Properties prop = new Properties();
 		InputStream input = null;
 
-		input = getClass().getClassLoader().getResourceAsStream(CallStatsConst.CallStatsJavaSDKPropertyFileName);
-		if (input != null) {
-			try {
-				prop.load(input);
-				BASE_URL = prop.getProperty("CallStats.BaseURL");
-				connectionTimeOut = Integer.parseInt(prop.getProperty("CallStats.ConnectionTimeOut"));
-				soTimeOut = Integer.parseInt(prop.getProperty("CallStats.SOTimeOut"));
-			} catch (IOException e) {
-				// TODO Auto-generated catch block
-				e.printStackTrace();
+		try {
+			input = new FileInputStream(CallStatsConst.CallStatsJavaSDKPropertyFileName);
+			if (input != null) {
+				try {
+					prop.load(input);
+					BASE_URL = prop.getProperty("CallStats.BaseURL");
+					connectionTimeOut = Integer.parseInt(prop.getProperty("CallStats.ConnectionTimeOut"));
+					soTimeOut = Integer.parseInt(prop.getProperty("CallStats.SOTimeOut"));
+				} catch (IOException e) {
+					// TODO Auto-generated catch block
+					e.printStackTrace();
+				}
 			}
+			
+		} catch (FileNotFoundException e1) {
```

Comment:
similar comments to the one below.
1. instead of using nested try...try...catch...catch, probably better to try...catch...catch...
2. probably better to fail fast and raise the exception up in the stack rather than continue running.